docs/en/latest/deployment-guide.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Deployment Guide
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Overview
+
+This document explains how to support multiple ways to deploy custom plugins.
+
+**Note: This feature is a WIP** 
```

Comment:
```suggestion
:::note

This feature is a WIP.

:::
```

Reply:
You can use this format, the effect is as follows
<img width="475" alt="image" src="https://user-images.githubusercontent.com/39793568/184048766-971d79e5-6618-4eba-bba8-8a21f46278ff.png">

Reply:
fixed, many thanks